### ISSUE TITLE
Which configuration does the '80.07' use?

### ISSUE THREAD

**Issue by YWMditto** [yeonsw/RankEncoder#2]
Hello! First, congratulations on your great job! 
I would like to ask what configuration is used for 80.07 in this table in the paper? Why is it different with the result in appendix, which also use sncse + rankencoder?

<img width="930" alt="image" src="https://user-images.githubusercontent.com/46778265/196719049-52c38185-c275-4d67-af71-273e19f7626c.png">
<img width="951" alt="image" src="https://user-images.githubusercontent.com/46778265/196719127-cbf60502-2849-42df-9290-68dcc83919f2.png">

Does the 80.07 use the same configuration with 79.6 +- 0.2 ?

Thanks for your reply!!!

**Comment by yeonsw** [comment]
Hi,

We used the SNCSE-bert-base model provided by the authors (https://github.com/Sense-GVT/SNCSE) for Table 1.
Table 2 is for measuring the standard deviation of each method.
Thus we had to trained SNCSE multiple times with different random seeds.
So, we trained SNCSE with three different random seeds and trained RankEncoder on these trained models.
In this setting, we used a smaller batch size for training SNCSE; the original setting of SNCSE uses a batch size of 256 but we used a batch size of 128.

Thanks
